src/notion/blocks.ts

```diff
@@ -30,6 +30,20 @@ export function paragraph(text: RichText[]): Block {
   };
 }
 
+export function toggle(
+  text: RichText[],
+  children: BlockWithoutChildren[] = []
```

Comment:
Toggles can nest indefinitely, `children` should just be an array of nomral blocks
![image](https://github.com/user-attachments/assets/8a598d3c-e033-43cd-8955-f5d454f72a06)

Reply:
I understand that the issue is that in gfm it's just html, and you can't nest that? Is that correct?